schemas/rdf/examples/Submodel_SubmodelElement_shortExample.ttl

```diff
@@ -1,66 +1,35 @@
-@prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
 
 <http://i40.customer.com/type/F13E8576F6488342>
-      a <https://admin-shell.io/aas/3/0/RC01/Submodel> ;
+      a <https://admin-shell.io/aas/3/0/RC02/Submodel> ;
       rdfs:label "Identification"^^xsd:string ;
-      <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "Identification"^^xsd:string ;
-      <https://admin-shell.io/aas/3/0/RC01/Referable/description> "Identification from Manufacturer"@en , "Hersteller-Identifikation"@de ;
+      <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "Identification"^^xsd:string ;
+      <https://admin-shell.io/aas/3/0/RC02/Referable/description> "Identification from Manufacturer"@en , "Hersteller-Identifikation"@de ;
       rdfs:comment "Identification from Manufacturer"@en ;
       rdfs:comment "Hersteller-Identifikation"@de ;
-      <https://admin-shell.io/aas/3/0/RC01/Submodel/submodelElement> [
-          a <https://admin-shell.io/aas/3/0/RC01/Property> ;
+      <https://admin-shell.io/aas/3/0/RC02/Submodel/submodelElement> [
+          a <https://admin-shell.io/aas/3/0/RC02/Property> ;
           rdf:subject <http://i40.customer.com/type/F13E8576F6488342/Manufacturer> ;
           rdfs:label "Manufacturer"^^xsd:string ;
-          <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "Manufacturer"^^xsd:string ;
-          <https://admin-shell.io/aas/3/0/RC01/Property/valueType>  "xsd:string"^^xsd:string ;
-          <https://admin-shell.io/aas/3/0/RC01/Property/valueId> [
-              a <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-              <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-                  a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-                  <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/KeyElements/GlobalReference> ;
-                  <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
-                  <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>
-              ] ;
+          <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "Manufacturer"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/valueType>  "xsd:string"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/valueId> [
+              a <https://admin-shell.io/aas/3/0/RC02/GlobalReference> ;
+              <https://admin-shell.io/aas/3/0/RC02/GlobalReference/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
           ] ;
-          <https://admin-shell.io/aas/3/0/RC01/Property/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
```

Comment:
The valueId is http://i40.customer.com/type/F13E8576F6488342/Manufacturer. So the value is <http://i40.customer.com/type/F13E8576F6488342/Manufacturer> or "X GmbH", isn'it it? It is very unprobable that the value is identical to the valueId

Reply:
Yes, you are right. I fixed the property/value.